invenio_vocabularies/cli.py

```diff
@@ -47,6 +51,51 @@ def _load_csv_data(path):
         return dicts
 
 
+def _create_languages_vocabulary(vocabulary_type_name, source_path):
+    """Load languages vocabulary."""
+    identity = Identity(1)
+    identity.provides.add(any_user)
+    service = VocabulariesService()
+
+    # Create vocabulary type
+    vocabulary_type = VocabularyType(name="languages")
+    db.session.add(vocabulary_type)
+    db.session.commit()
+
+    # Load data
+    records = []
+    instance_languages = [lang[0] for lang in current_i18n.get_languages()]
+
+    for language in pycountry.languages:
+        lang_code = language.alpha_3
+        titles = {}
+        try:
+            l = Locale.parse(lang_code)
+            titles = {}
+            for lang in instance_languages:
+                titles[lang] = l.get_display_name(lang)
+            # Create record
+            record = service.create(
+                identity=identity,
+                data={
+                    "metadata": {
+                        "id": lang_code,
+                        "title": titles,
+                    },
+                    "vocabulary_type_id": vocabulary_type.id,
+                    "vocabulary_type": vocabulary_type.name
+                },
+            )
+            records.append({
+                "id": lang_code,
+                "title": titles,
+            })
```

Comment:
minor: simplification (indentation is far from great when coding in a suggestion so don't commit as is)
```suggestion
            # Create record
            metadata =  {
                        "id": lang_code,
                        "title": titles,
                    }
            record = service.create(
                identity=identity,
                data={
                    "metadata": metadata,
                    "vocabulary_type_id": vocabulary_type.id,
                    "vocabulary_type": vocabulary_type.name
                },
            )
            records.append(metadata)
```